app/service/rest_svc.py

```diff
@@ -171,14 +171,14 @@ async def create_operation(self, access, data):
 
     async def create_schedule(self, access, data):
         operation = await self._build_operation_object(access, data['operation'])
-        schedules = await self.get_service('data_svc').locate('schedules', match=dict(name=operation.name))
+        schedule_id = str(uuid.uuid4())
+        schedules = await self.get_service('data_svc').locate('schedules', match=dict(id=schedule_id))
         if schedules:
-            self.log.debug('A scheduled operation with the name "%s" already exists, skipping' % operation.name)
+            self.log.debug('A scheduled operation with the id "%s" already exists, skipping' % schedule_id)
```

Comment:
Let's change this to "A schedule with the id X already exists, skipping" to avoid making it sound like the ID is for the operation